index.js

```diff
@@ -1,48 +1,49 @@
 var express = require('express');
 var app = express();
 var bodyParser = require('body-parser');
-var data = require('./data');
 var cors = require('cors');
+import connect from './util/db';
 
-app.use(cors());
+import {
+  getAllFarms,
+  getFarmsByZip,
+  getFarmsByState,
+} from './controllers/farm.controller'
 
-app.get('/', function (req, res) {
-  res.send('Hello World!');
-});
+import {
+  cleanUpStateNames,
+} from './util/actions';
 
-app.get('/api', function (req, res) {
-  res.send(data);
-});
+//---
+// Routes
+//
 
-app.get('/api/zip/:zipcode', function (req, res) {
-  var response = [];
-  data.forEach(function(item, index) {
-     if(item.Location_Zip == req.params.zipcode) {
-       response.push(item);
-     }
-  })
+// Index route
 
-  res.send(response);
-  console.log('found ' + response.length + ' items');
+app.get('/', function (req, res) {
+  res.send('👋');
 });
 
-app.get('/api/state/:name', function (req, res) {
-  var response = [];
-  data.forEach(function(item, index) {
-     if(item.Location_State.toLowerCase() == req.params.name.toLowerCase()) {
-       response.push(item);
-     }
-  })
+// Farm routes
 
-  res.send(response);
-  console.log('found ' + response.length + ' items');
-});
+app.get('/api', getAllFarms);
+
+app.get('/api/zip/:zipcode', getFarmsByZip);
+
+app.get('/api/state/:state', getFarmsByState);
 
-// Body parser
-app.use(bodyParser.json());
+// Util routes for development help
 
+app.get('/tools/cleanUpStateNames/:name', cleanUpStateNames);
 
+//---
+// Connect to Database and boot server
 
-app.listen(3000, function () {
-  console.log('Example app listening on port 3000!');
+connect()
+.then(() => {
+  app.use(cors());
+  app.use(bodyParser.json());
```

Comment:
You can move these two middleware out of here, you only really care about delaying the app startup until you have your database connection